### PR TITLE
PR template for adding packages

### PR DESCRIPTION
**Pull request by grst**
This adds a PR template with a questionnaire for new tools. 

Right now, this is partly redundant to the metadata that will go into the yaml file, so maybe we should reduce the number of questions.

### CODE REVIEW

File: .github/pull_request_template.md
Comment (grst) [inline .github/pull_request_template.md:28]: This one is stolen from the [AIRR software checklist](https://docs.airr-community.org/en/stable/swtools/airr_swtools_checklist.html#toolschecklist). I like the idea because it forces package authors to at least think about it. 

I'm not sure if we should also advertise the scverse dicsoure as a good place for this here?

File: .github/pull_request_template.md
Comment (grst) [inline .github/pull_request_template.md:36]: Do we want something like this? If we do, we should also get approval for all packages we add ourselves in the beginning. 

On conda-forge they require the package maintainer to agree. This is usually solved by pinging them on github.

**Review by mikelkou — APPROVED** [review]

**Comment by grst** [comment]
I decided it's better to have a minimalistic PR template that just refers to the README. Otherwise we have annoying redundancies. 

The new PR template is not part of #1.

**Pull request closed without merging**
